### ISSUE TITLE
[Request] auto-cleaning players

### ISSUE THREAD

**Issue by Nord1cWarr1or** [OciXCrom/RankSystem#4]
I request for auto cleaning players, were not active longer than N days.

**Comment by OciXCrom** [comment]
Good idea. I'll add this in the next update. If you don't want to wait, you can add it yourself. It's just one line of code.

Find the line that contains `nvault_open` and add this directly under it:

```pawn
nvault_prune(g_iVault, 0, get_systime() - (60 * 60 * 24 * N_DAYS));
```

Replace `N_DAYS` with the number of days you want.

**Comment by Nord1cWarr1or** [comment]
> Find the line that contains `nvault_open` and add this directly under it:
> 
> ```pawn
> nvault_prune(g_iVault, 0, get_systime() - (60 * 60 * 24 * N_DAYS));
> ```
> 
> Replace `N_DAYS` with the number of days you want.

I mean all it for MySQL method of save
